README.md

```diff
@@ -1 +1,75 @@
 # weather-app
+![Linters](https://github.com/descholar-ceo/weather-app/workflows/Linters/badge.svg)
+
+## Project description
+This project is an implementation of a weather app.
+
+This project was done mainly for learning purposes, the aim was to practice using javascript async/await and working with APIs using fetch method. The project's specifications are listed on [The odin project](https://www.theodinproject.com/courses/javascript/lessons/weather-app)
+
+## Screenshot
+![demo picture](./src/assets/images/weather-app-screenshot.png)
+
+## Built with
+- HTML5
+- CSS3
+- Vanilla Javascript
+- Webpack
+
+## Features
+  - Search the weather info of any city on the earth
+  - Convert the gotten result from degrees celcius to fahreneit
+
+## Deployment
+- [Wearher app on netlify](https://weather-app-descholar.netlify.app)
+- [Wearher app on heroku](https://weather-app-descholar.heroku.com)
+
```

Comment:
- [ ] Kindly correct the typo here. It should be `Weather app` instead.

Reply:
Resolved